src/SubComponents/Notification.js

```diff
@@ -27,11 +29,35 @@ export const Notification = ({ children, ...otherProps }) => {
       }}
       {...otherProps}
     >
-      {children}
+      {(t) => (
+        <ToastBar toast={t}>
+          {({ icon, message }) => (
+            <>
+              {icon}
+              {message}
+              <XIcon
+                className="plc-w-10 plc-h-5 plc-cursor-pointer"
+                onClick={() => toast.dismiss(t.id)}
+              />
+            </>
+          )}
+        </ToastBar>
+      )}
     </Toaster>
   );
 };
 
 Notification.viewId = "notification";
 
+toast.warning = (msg, options) =>
```

Comment:
toast doesn't ship with a `warning` method, it only has `success` and `error`, so I extended it by adding this method